sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_v1/query.sql

```diff
@@ -4,6 +4,7 @@ WITH combined AS (
     DATE(submission_timestamp) AS submission_date,
     'suggest' AS source,
     'impression' AS event_type,
+    'desktop' AS form_factor,
```

Comment:
We introduce a new field here and call it `form_factor` to match the interface we use when streaming to adM. I'll backfill the existing data to set `form_factor` to "desktop" for all existing rows.